src/main/java/emissary/util/xml/JDOMUtil.java

```diff
@@ -267,7 +267,7 @@ public static int getChildIntValue(final Element el, final String childName) {
         try {
             x = Integer.parseInt(val);
         } catch (NumberFormatException ex) {
-            logger.debug("Unable to parse {} as an integer", val);
+            logger.debug("Unable to parse String as an integer");
```

Comment:
Should these be error since it is an exception?
Also is it worth logging the ex message?

Reply:
These were too noisy and changed to debug.